crypto/keys/keybase_test.go

```diff
@@ -94,6 +116,24 @@ func TestKeyManagement(t *testing.T) {
 	// make the storage with reasonable defaults
 	cstore := NewInMemory()
 
+	// Test modified supported algos
+	supportedAlgos := cstore.SupportedAlgos()
+	secpSupported := false
+	edSupported := false
+	srSupported := false
+	for _, supportedAlgo := range supportedAlgos {
+		if Secp256k1 == supportedAlgo {
```

Comment:
This can be solved with something like:

```golang
switch {
case Secp256k1 == supportedAlgo:
	// snip
supportedAlgo == Ed25519:
	// snip
supportedAlgo == Sr25519:
	// snip
}
```

Reply:
Thanks!